README.md: add NewWithAPIToken usage (#763)
cloudflare/cloudflare-go: README.md

)

func main() {

> 	// Construct a new API object with API KEY

Reviewer: ```suggestion
	// Construct a new API object using a global API key
```

	api, err := cloudflare.New(os.Getenv("CLOUDFLARE_API_KEY"), os.Getenv("CLOUDFLARE_API_EMAIL"))

> 	// alternatively, you can use an API TOKEN
> 	//api, err := cloudflare.NewWithAPIToken(os.Getenv("CLOUDFLARE_API_TOKEN"))

Reviewer: ```suggestion
	// alternatively, you can use a scoped API token
	// 
	// api, err := cloudflare.NewWithAPIToken(os.Getenv("CLOUDFLARE_API_TOKEN"))
```

)

func main() {
	// Construct a new API object with API KEY
	api, err := cloudflare.New(os.Getenv("CLOUDFLARE_API_KEY"), os.Getenv("CLOUDFLARE_API_EMAIL"))
	// alternatively, you can use a scoped API token

> 	// 

Reviewer: ```suggestion
```
